src/blockchain/deposit_strategy/gas_price_calculator.py

```diff
@@ -62,7 +62,6 @@ def calculate_deposit_recommendation(self, deposit_strategy: BaseDepositStrategy
             )
             base_fee_per_gas = self._get_pending_base_fee()
             success = recommended_max_gas >= base_fee_per_gas
-            logger.info({'msg': 'Calculations deposit recommendations.', 'value': success})
```

Comment:
Why log removed?

Reply:
This was a duplicated log https://github.com/lidofinance/depositor-bot/blob/73016b8bfc45abd4a39d505374499e0acf0a7f10/src/bots/depositor.py#L193